Add files via upload

Harshdev625 opened the pull request.

Sapna127 commented:
Please update the readme file and add the algorithm of the problem along with TC and SC.

Harshdev625 commented:
<!--
/* Font Definitions */
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
/* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin:0cm;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:blue;
	text-decoration:underline;}
.MsoChpDefault
	{mso-style-type:export-only;}
@page WordSection1
	{size:612.0pt 792.0pt;
	margin:72.0pt 72.0pt 72.0pt 72.0pt;}
div.WordSection1
	{page:WordSection1;}
-->Ma’am,            I have updated Readme and also added SC and TC in my code. Please check and let me know whether further changes are required or if it’s completely fine. Sent from Mail for Windows From: Sapna KulSent: 12 October 2022 00:57To: Sapna127/CP-LiteCc: HARSH DEV; AuthorSubject: Re: [Sapna127/CP-Lite] Add files via upload (PR #11) Please update the readme file and add the algorithm of the problem along with TC and SC.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***> 

Harshdev625 commented:
Ma'am Updated.

Harshdev625 commented:
Please Merge it.
